angr/procedures/win32/strings.py

```diff
@@ -0,0 +1,22 @@
+import angr
+from angr.sim_type import SimTypeString, SimTypeInt
```

Comment:
Since `procedures/win32/strings.py` is part of angr, I think it's better use a relative import instead of `import angr` here. We did `import angr` for many other SimProcedures, but I think that is bad, and I do not remember why we did that.

Reply:
I'm almost certain it's `import angr` because of the paradigm that simprocedures should be able to live either in-tree or out-of-tree.

Reply:
Are we going to copy those SimProcedures out of angr in the near future?

Reply:
no

Reply:
but like it would be nice if all simprocedure files just looked similar

Reply:
_welp_

Fine.

Reply:
Does this PR look good to you? I'm about to click "merge".